MetadataExtractor/Formats/Apple/BplistReader.cs

```diff
@@ -66,7 +66,7 @@ public static PropertyListResults Parse(byte[] bplist)
 
```

Comment:
The list declaration just above here could define a capacity of `offsets.Length`.